### ISSUE TITLE
What's up with the status on Dark Mode? How is the Product Backlog priority be managed? Love the product - this feature is crucial for future success. Thanks! 

### ISSUE THREAD

**Issue by locolucco209** [brave/browser-laptop#14690]
### Steps to Reproduce

  1.
  2.
  3.

**Actual result:**

**Expected result:**

**Reproduces how often:**

### Brave Version

**about:brave info:**

**Reproducible on current live release:**

**Comment by bsclifton** [comment]
Hi @locolucco209 - we have currently captured this issue with https://github.com/brave/browser-laptop/issues/271 so I'll close this one as a dupe

As for our priorities:
- Dark mode is currently not prioritized for our current version (Muon)
- We are moving to a Chromium based UI (Brave Core) in the next few months which has support for themes
- We have already tested a dark mode theme with Brave Core and it looks great!

Long story short- give us a few months and we should have something for you 😄👍 Thanks for the feedback, glad you love the product! 🎉

**Comment by nikisix** [comment]
Can't wait.. is it in beta per-chance?

**Comment by bsclifton** [comment]
@nikisix unfortunately not yet- but you can get more information here:
https://github.com/brave/brave-browser

And here's a link to the themes:
https://github.com/brave/brave-chromium-themes
